### ISSUE TITLE
Inconsistent API error responses.

### ISSUE THREAD

**Issue by Steve-Walsh** [wit-ai/wit#1999]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
Bug

**What is the current behavior?**
When trying to create an intent via the API, I get an error message back saying. I've seen the same behaviour across the delete intent, create utterance and delete utterance endpoints
.
``` json
{
    "error": "Something went wrong. We've been notified",
    "code": "unknown"
}
```
If I hit the API with the same request a second time I get the correct error message. 

```json
{
    "error": "\"intentName\" already exists",
    "code": "bad-request"
}
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Make this request twice 
```
curl --location --request POST 'https://api.wit.ai/intents?v=20200513' \
--header 'Authorization: Bearer APIKEY' \
--header 'Content-Type: application/json' \
--data-raw '{"name": "intentName"}'
```
**What is the expected behavior?**
The correct error message is returned the first time. 

**If applicable, what is the App ID where you are experiencing this issue? If you do not provide this, we cannot help.**
143708370798886

**Comment by avoinkov** [comment]
I got the same random error multiple times. Today I've got another error on request to create intent. I've not created separate post cause I believe these errors are of random type and not easy to reproduce by doing certain steps.
BTW if somebody is going to check the cause here is info i have:
APP ID: f0d3a69c-0ed7-46be-85b5-04eae4a7199b (is deleted)
REQUEST DATA:
```
{
  url: 'https://api.wit.ai/intents?v=20200513',
  request: {
    method: 'post',
    body: '{"name":"buy_product_by_full_description"}',
    headers: {
      Authorization: 'Bearer TOKEN',
      'Content-Type': 'application/json'
    }
  }
}
```

RESPONSE DATA:
status: 400,
statusText: 'Bad Request',
`{ error: 'Parser not found', code: 'not-found' }`
HINT: It may be related with the fact that we create Wit.ai app and in a few seconds start creating intents/entities
